README.md

```diff
@@ -1 +1,18 @@
-Install the [YoWASP VS Code extension](https://marketplace.visualstudio.com/items?itemName=yowasp.toolchain), then run the "YoWASP Toolchain: Build..." command.
\ No newline at end of file
+Install the [YoWASP VS Code extension](https://marketplace.visualstudio.com/items?itemName=yowasp.toolchain), then run the "YoWASP Toolchain: Build..." command for those unfamiliar with visual studio code pressing `Ctrl + Shift + P` and then typing `YoWASP` until you see an option is how you do that. There is also a "YoWASP Toolchain: Connect to USB Device..." command which can but useful to see whether your USB device is detected without rebuilding things.
+
+
+# This is the first time i am using Yosys for this board (in a while)
+The repository is currently configured for the [ulx3s](https://radiona.org/ulx3s/) board. Modify `.vscode/settings.json` to suit your model. Here are some useful resources:
+
+- [List of supported openFPGALoader boards](https://trabucayre.github.io/openFPGALoader/compatibility/board.html)
+- [The YoWASP website has a "How do I use the tools" section which can give you an idea what packages you might want to look into](https://yowasp.org/)
+- Make files of Yosys projects for your target. These obviously are different for your target but can give you good idea what flags you might need to use
+- For a board from a different family of FPGA, you probably need to change these lines:
+  - `"@yowasp/nextpnr-ecp5",` needs to be changes to reflect your FPGA type.

```

Comment:
I've updated the `main` branch; you no longer need to modify the list of packages since all of the available YoWASP packages are included by default.